src/TwigYard/Middleware/Error/ErrorMiddleware.php

```diff
@@ -123,16 +138,23 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
      */
     private function getErrorPageStream(string $pageName): Stream
     {
-        $errPage = $this->appState->getSiteDir() . '/' . $this->templateDir . '/' . $pageName;
-        $localeErrPage = $this->appState->getSiteDir()
-            . '/' . $this->templateDir
-            . '/' . $this->appState->getLocale()
-            . '/' . $pageName;
+        $errPage = '';
+        $localeErrPage = '';
+
+        try {
+            $errPage = $this->appState->getSiteDir() . '/' . $this->templateDir . '/' . $pageName;
+            $localeErrPage = $this->appState->getSiteDir()
+                . '/' . $this->templateDir
+                . '/' . $this->appState->getLocale()
+                . '/' . $pageName;
+        } catch (\TypeError $typeError) {
+            new Stream(__DIR__ . '/' . $pageName);
```

Comment:
I think we can leave this `catch` block empty. This wont really do anything as it is missing the `return` statement.